### PR TITLE
Update metrics.asciidoc

### PR DESCRIPTION
**Pull request by leahleahy**

### CODE REVIEW

File: docs/metrics.asciidoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [[metrics]]
 == Metrics
 
-The Go agent periodically gathers and reports metrics. You can control how
+The Go agent periodically gathers and reports metrics. Control how
```
Comment (EamonnTP) [inline docs/metrics.asciidoc:4]: Go {agent}?

File: docs/metrics.asciidoc [hunk 2]
```diff
@@ -81,7 +81,7 @@ The total virtual memory the process has.
 [[metrics-golang]]
 === Go runtime metrics
 
-The Go agent reports various Go runtime metrics.
+The Go {agent} reports various Go runtime metrics.
```
Comment (EamonnTP) [inline docs/metrics.asciidoc:84]: I would double check to ensure that the correct Terminology is Go Elastic Agent. If so, you can leave as.

**Review by EamonnTP — CHANGES_REQUESTED** [review]
Looks good. Just a couple of suggestions.

File: docs/metrics.asciidoc (hunk 1 above)
Comment (bmorelli25) [reply docs/metrics.asciidoc:4]: @leahleahy, `{agent}` resolves to `Elastic Agent`, which is different from the APM agents. In the APM agent docs, we use just the word, `agent`.

File: docs/metrics.asciidoc (hunk 2 above)
Comment (bmorelli25) [reply docs/metrics.asciidoc:84]:
```suggestion
The Go agent reports various Go runtime metrics.
```

File: docs/metrics.asciidoc [hunk 3]
```diff
@@ -279,10 +279,10 @@ type: long
 format: count (delta)
 
 The number of transactions for which breakdown metrics (`span.self_time`) have been created.
-As the Go agent tracks the breakdown for both sampled and non-sampled transactions, this
+As the Go {agent} tracks the breakdown for both sampled and non-sampled transactions, this
```
Comment (bmorelli25) [inline docs/metrics.asciidoc:282]:
```suggestion
As the Go agent tracks the breakdown for both sampled and non-sampled transactions, this
```

File: docs/metrics.asciidoc (hunk 1 above)
Comment (leahleahy) [reply docs/metrics.asciidoc:4]: Got it, thank you!

**Comment by bmorelli25** [comment]
Using the "edit" functionality to change branches is unfortunately too messy. I've cherry-picked the commits in this PR to a new branch and opened https://github.com/elastic/apm-agent-go/pull/951.

**Pull request closed without merging**